CHANGELOG.md

```diff
@@ -91,7 +95,7 @@ This is a Lotus Node only release, which includes a variety of new features, imp
 
 ## 📝 Changelog
 
-See https://github.com/filecoin-project/lotus/compare/v1.28.2...release/v1.29.0 for the set of changes since the last release.  
```

Comment:
lots of trailing newline fixes in this diff, did you mean to do this? I don't mind too much but it is noisy if you weren't intending to and your editor was just being clever.

Reply:
I think this is ZED.

Reply:
Will look into how to avoid these, dont know why it's doing this.